javascript/javascript3/week3/visual/main.js

```diff
@@ -0,0 +1,30 @@
+moveElement(document.querySelector("li"), { x: 20, y: 300 }).then(() => {
+  console.log("Element has been moved");
+});
+
+async function translateOneByOne() {
```

Comment:
Don't forget to call this method if you want to use it 🙂 

How would you go about implementing the other method? Where it translates all at once? Doesn't that mean you're simply not `awaiting` the first circle to finish moving before starting the next one? 😉 😉